.gitignore

```diff
@@ -0,0 +1 @@
+**/tags
```

Comment:
Editor-specific ignore rules go in your [global .gitignore file](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore), not in every project you contribute to.

Reply:
Removed it.